ux.symfony.com/templates/base.html.twig

```diff
@@ -35,7 +35,7 @@
             {% endblock %}
         {% endblock %}
     </head>
-    <body>
+    <body data-turbo="false">
```

Comment:
No turbo?

Reply:
It messed with my local, we could try but i'm more and more for "either use Turbo or remove Turbo" :|

Reply:
I don't understand. I say "yes, let's use Turbo", which we are now. So why add this?

Reply:
Oh sorry forgot to remove this one !!

Reply:
(or added it back on rebase)

But nothing volontarely ... that would break the site ^^

Reply:
Fixed sorry